src/plugins/data/public/search/session/sessions_mgmt/components/table/table.test.tsx

```diff
@@ -175,13 +175,15 @@ describe('Background Search Session Management Table', () => {
             />
           </LocaleWrapper>
         );
-        jest.advanceTimersByTime(20000);
-      });
-
-      // 1 for initial load + 2 refresh calls
-      expect(sessionsClient.find).toBeCalledTimes(3);
 
-      jest.useRealTimers();
+        await waitFor(
+          () => {
+            // 1 for initial load + 2 refresh calls
+            expect(find).toHaveBeenCalledTimes(3);
+          },
+          { timeout: 10000, interval: 100 }
```

Comment:
I think the intent of this test, originally, was to make sure the `refreshInterval` in the management config was utilized. Now that we're setting it to 1s and waiting up to 10s for it to refresh twice, I'm not sure the original intent is preserved.

TBH though, I find this a difficult thing to really test well, and it may be enough just to ensure `find` is called just twice (once for the initial fetch & once for re-fetch).

Reply:
you're right, the test is labeled `re-fetches data` and so just to check that it's called twice should be sufficient. It was hard to fix the test, it passed locally, but not in the cloud, applied changes, same ... so I'm not sure how my latest changes will do , applied suggestions, keeping 🤞 it works